lib/index.js

```diff
@@ -1126,6 +1130,85 @@ class Mcdev {
         Util.logger.info(`:: Done\n`);
         return result;
     }
+    /**
+     * Updates notification email address field
+     *
+     * @param {string} businessUnit name of BU
+     * @param {TYPE.SupportedMetadataTypes} type limit execution to given metadata type
+     * @param {string[]} [keys] customerkey of the metadata
+     * @returns {Promise.<Object.<string, string[]>>} key: business unit name, value: list of affected item keys
+     */
+    static async updateNotifications(businessUnit, type, keys) {
```

Comment:
- [ ] lets please include this in `Mcdev.#runMethod` like I did for `Mcdev.fixKeys`.